flink-table/flink-table-common/src/main/java/org/apache/flink/table/functions/BuiltInFunctionDefinitions.java

```diff
@@ -433,7 +433,7 @@ ANY, and(logical(LogicalTypeRoot.BOOLEAN), LITERAL)
 
     public static final BuiltInFunctionDefinition URL_DECODE =
             BuiltInFunctionDefinition.newBuilder()
-                    .name("url_decode")
+                    .name("URL_DECODE")
                     .kind(SCALAR)
                     .inputTypeStrategy(sequence(logical(LogicalTypeFamily.CHARACTER_STRING)))
                     .outputTypeStrategy(nullableIfArgs(explicit(DataTypes.STRING().nullable())))
```

Comment:
@snuyanzin As I saw this pr and realized I missed your message on the previous one(#24773).
For the output type strategy, I think it should be 'outputTypeStrategy(explicit(DataTypes.STRING().nullable()))' because the return value may be null even if the input is non-null (e.g., decoding failure), otherwise the downstream operator may trust the non-null property and encounter a runtime exception (e.g., NPE) if skip the null check for field accessing. 
Does this make sense to you?

Reply:
@lincoln-lil thanks for the feedback

now I see what you mean
Yep, it makes, thanks for clarification

I updated the PR to address this, could you please approve/reject?